### DISCUSSION TITLE
Remove client.0cca1885.js from output files

### DISCUSSION THREAD

**Discussion by jylin** [withastro/roadmap#242] [Proposal]
This file seems to be created in the dist directory but not referenced from any of the other files in the build output. What does it do  and could it be removed if it does nothing?

**Comment by kylebutts** [comment]
@jylin Are you using a js framework with react/vue/etc.? When you add those frameworks, they have to ship the client side javascript for that framework. It won't be included in any of the html files if you don't actually use a client-side js component

**Reply by jylin** [reply]
Yea, using React and that makes sense. Thanks for the explanation. I guess it would be a minor improvement then to not generate that file at all if no js use is detected.
